src/site/content/en/installable/add-manifest/index.md

```diff
@@ -19,9 +19,9 @@ launched.
 Manifest files are supported in Chrome, Edge, Firefox, UC Browser, Opera and the
 Samsung browser, and Safari is currently working on adding support.
 
-## Create the manifest.json file
+## Create the manifest.webmanifest file
```

Comment:
@petele let us know if this is indeed the convention now

Reply:
I see now that @AdamDemirel did indeed link to the spec, which backs up his request: https://www.w3.org/TR/appmanifest/#using-a-link-element-to-link-to-a-manifest